src/python/src/grpc/_adapter/_low_test.py

```diff
@@ -115,6 +115,12 @@ def tearDown(self):
   def _perform_echo_test(self, test_data):
     method = 'test method'
     details = 'test details'
+    server_leading_metadata_key = 'my_server_leading_key'
+    server_leading_metadata_value = 'my_server_leading_value'
```

Comment:
I don't know if it's documented, but it's true: https://github.com/grpc/grpc/blob/2e165e34010e44f20c6ecea63b20a6996810313f/src/core/transport/chttp2/bin_encoder.c#L276